web/client/reducers/__tests__/draw-test.js

```diff
@@ -86,6 +85,155 @@ describe('Test the draw reducer', () => {
         expect(state.tempFeatures).toExist();
         expect(state.tempFeatures[0]).toBe(feature);
     });
+
+    it('POLYGON GEOMETRY_CHANGED', () => {
+        let feature1 = {
+            type: "Feature",
+            geometry: {
+                type: "Polygon",
+                coordinates: [
+                    [[35.0, 10.0], [45.0, 45.0], [15.0, 40.0], [10.0, 20.0], [35.0, 10.0]],
+                    [[20.0, 30.0], [35.0, 35.0], [30.0, 20.0], [20.0, 30.0]]
+                ]
+
+            },
+            id: 'idFt1',
+            properties: {
+                someProp: "someValue"
+            }
+        };
+        let testAction = {
+            type: GEOMETRY_CHANGED,
+            features: [feature1]
+        };
+        let state = draw({}, testAction);
+        expect(state.tempFeatures).toExist();
+        expect(state.tempFeatures[0]).toBe(feature1);
+        expect(state.tempFeatures.length).toEqual(1);
+    });
+    it('Point GEOMETRY_CHANGED', () => {
+        let feature2 = {
+
+            type: "Feature",
+            geometry: {
+                type: "Point",
+                coordinates: [1, 2]
+            },
+            id: 'idFt2',
+            properties: {
+                someProp: "someValue2"
+            }
+        };
+        let testAction = {
+            type: GEOMETRY_CHANGED,
+            features: [feature2]
+        };
+        let state = draw({}, testAction);
+        expect(state.tempFeatures).toExist();
+        expect(state.tempFeatures[0]).toBe(feature2);
+        expect(state.tempFeatures.length).toEqual(1);
+    });
+    it('MultiPolygon GEOMETRY_CHANGED', () => {
+        let feature3 = {
+            type: "Feature",
+            geometry: {
+                type: "MultiPolygon",
+                coordinates: [
+                    [
+                        [ [102.0, 2.0], [103.0, 2.0], [103.0, 3.0], [102.0, 3.0], [102.0, 2.0] ]
```

Comment:
please use values that are not normalized so we test everything works fine
for example
here

```suggestion
                        [ [-202.0, 2.0], [-303.0, 2.0], [-103.0, 3.0], [302.0, 3.0], [182.0, 2.0] ]
```